src/components/include/cmap.h

```diff
@@ -121,43 +146,42 @@ cmap_add(cmap_t *m, void *val)
 	/* no free slots? Create more! */
 	if (free == -1) {
 		long lower, upper;
-		if (val != CVECT_INIT_VAL && 
-		    0 > cvect_add_id(m->data, val, cmap_to_vect_id(m->id_boundary))) return -1;	
-		free = lower = m->id_boundary;
+		if (val != CVECT_INIT_VAL && 0 > cvect_add_id(m->data, val, cmap_to_vect_id(m->id_boundary))) return -1;
+		free = lower   = m->id_boundary;
 		m->id_boundary = upper = lower + CVECT_BASE;
 		/* Add the new values to the free list */
 		while (lower != upper) {
 			int idx = cmap_to_vect_freeid(lower);
-			if (cvect_add(m->data, (void*)(lower+1), idx)) assert(0);
+			if (cvect_add(m->data, (void *)(lower + 1), idx)) assert(0);
 			lower++;
 		}
 		/* The end of the freelist */
-		if (__cvect_set(m->data, cmap_to_vect_freeid(upper-1), (void*)-1)) assert(0);
+		if (__cvect_set(m->data, cmap_to_vect_freeid(upper - 1), (void *)-1)) assert(0);
 	}
 
 	is = __cvect_lookup(m->data, cmap_to_vect_id(free));
 	assert(NULL != is);
 	is->c.val = val;
 
-	is_free = cos_val_to_free(is);
-	m->free_list = (long)is_free->c.val;
-	is_free->c.val = (void*)-1;
+	is_free        = cos_val_to_free(is);
+	m->free_list   = (long)is_free->c.val;
+	is_free->c.val = (void *)-1;
 
 	return free;
 }
 
-/* 
+/*
  * This function will try to find an empty slot specifically for the
  * identifier id, or fail.
  *
  * This is O(n), n = MAX_EVENTS.  If we make the cvect_intern
  * have a doubly linked free list, this would go away, but I don't
- * want the memory consumption.  
+ * want the memory consumption.
  *
  * If you don't O(n), then just use a vect, not a map.  Map is for
  * cmap_add.
  */
-static inline long 
+static inline long
```

Comment:
does it fix function prototypes when the type and modifiers are on the *same* line as the function name?  Haven't seen an instances of that, so I'm skeptical.